app/assets/stylesheets/about.scss

```diff
@@ -0,0 +1,116 @@
+// Place all the styles related to the aboutme controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+.name 
+h1 {
+    font-size: 150px ;
+    text-align:center;
+    font-family:fantasy;
+    padding-top:50px;
+    position: relative;
+    margin: 0px 0px 0px 0px;
+    padding: 0px 150px 0px 150px;
+}
+
+.name {
+    display:inline-flex;
+    justify-content: center;
+    background-color:#55a8c9; 
+    flex-wrap:nowrap; 
+    width:100%
+    
+}
+
+.name img{
+    padding-right:0px;
+    margin-right:0px;
+    padding-top:75px;  
+}
+
+.divider
+p {
+    font-size: 50px;
+    text-align:center;
+    padding-top:0px;
+    background-color:#3AAFA9;
+    color:white;
+    margin: 0;
+    padding: 0;
+    font-family:Georgia, 'Times New Roman', Times, serif;
+}
+
+.media {
+    display: inline-flex;
+    justify-content: space-around;
+    flex-direction: row;
+    width:100%;
+    background-color:bisque;
+    margin: 0;
+    padding: 0;
```

Comment:
Out of curiosity, I see you're specifying margin and padding to be 0 in a lot of places. Is this necessary? If so, could there be a different way to achieve the same thing, such as setting margin and padding to 0 at a higher level?

Reply:
Good idea, I made a new class and added margin and padding to it, and applied it at a higher level